.kitchen.yml

```diff
@@ -60,6 +64,31 @@ suites:
           checkpoint: true
           switch_user: yes
 
+- name: attributes
```

Comment:
I am in favor of not using any fancy YAML stuff at all as it's not typically used for .kitchen files as far as I can tell.  It's a test file so it should be pretty explicit for each suite what values are being set and if it's repetitive I don't see that as an issue.